src/ibm/hypervisor-network-mgr-src/hyp_network_manager.cpp

```diff
@@ -187,7 +187,10 @@ void HypNetworkMgr::setBIOSTableAttrs()
             {
                 const std::string& itemType =
                     std::get<biosBaseAttrType>(item.second);
-
+                if (item.first.rfind("vmi_if1", 0) == 0)
```

Comment:
need to check for std::string::npos?
if (item.first.rfind("vmi_if1", 0) != std::string::npos)?

Reply:
we can use if (item.first.rfind("vmi_if1") != std::string::npos)

but all attributes for if1 start with "vmi_if1" , that's why I am check if vmi_if1 found at pos 0 .